urllib3/fields.py

```diff
@@ -130,7 +130,7 @@ def _render_parts(self, header_parts):
             iterable = header_parts.items()
 
         for name, value in iterable:
-            if value:
+            if value or value == '':
```

Comment:
I think presumably what we want to do is to exclude `None` as a value. So let's change this check to `is not None` instead.